### PR TITLE
Dblclick automation refactoring (#239)

### PR DESCRIPTION
**Pull request by helen-dikareva**
\cc @AlexanderMoskovkin @AndreyBelym 
Dbl click uses click automation, but click wait for new version of Hammerhead (coming soon). It'll not be a problem refactor click after this PR.

### CODE REVIEW

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

File: src/client/runner/automation/playback/dblclick.js [hunk 1]
```diff
@@ -5,201 +5,206 @@ import { fromPoint as getElementFromPoint } from '../get-element';
...
+        // NOTE: If a target element is a contentEditable element, we need to call focusAndSetSelection directly for
+        // this element. Otherwise, if the element obtained by elementFromPoint is a child of the contentEditable
+        // element, a selection position may be calculated incorrectly (by using the caretPos option).
+        var elementForFocus = domUtils.isContentEditableElement(this.element) ?
+                              this.element : this.eventArgs.element;
+
+        return automationUtil.focusAndSetSelection(elementForFocus, this.eventState.simulateFocus, this.caretPos);
+    }
+
+    _mouseup () {
```
Comment (AlexanderMoskovkin) [inline src/client/runner/automation/playback/dblclick.js]: `clickAutomation` with have the same `_focus, _mousedown, _mouseup, _click` methods, so it's code duplication. Let's use `clickAutomation` for the second click too

**Comment by helen-dikareva** [comment]
Ok, so wait for my refactoring clickAutomation and FPR commit

**Comment by helen-dikareva** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm
@VasilyStrelyaev please check the strings

File: src/client/runner/automation/playback/dblclick.js [hunk 2]
```diff
@@ -1,233 +1,122 @@
...
+                this.eventArgs            = clickAutomation.eventArgs;
+
+                if (browserUtils.isIE)
+                    eventUtils.unbind(document, 'focus', eventUtils.preventDefault, true);
+            });
+    }
+
+    _dblClick () {
+        // NOTE: If the element under the cursor has changed after second
+        // 'mousedown' event then we should not raise 'dblclick' event
```
Comment (VasilyStrelyaev) [inline src/client/runner/automation/playback/dblclick.js]:
```
        // NOTE: If an element under the cursor has changed after the second
        // 'mousedown' event, we should not raise the 'dblclick' event
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by helen-dikareva** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AndreyBelym** [comment]
lgtm

**Pull request merged**